### PR TITLE
Changes Tradeband syllables.

### PR DESCRIPTION
**Pull request by Belsima**
Ported from Paradise. I don't mind less serious syllables for languages, but hearing a supposedly unknown language that's actually an irl language that should be completely unrelated that there's classes for in my school 1 room across from my French class doesn't seem right.

### CODE REVIEW

**Comment by Anewbe** [comment]
Instead of removing the old syllables, mix in the new ones? Tradeband sounding imperious is very much a good thing, in my opinion.

**Comment by Belsima** [comment]
That could work, I suppose.

**Comment by Belsima** [comment]
Ah- wait. No. Not really. The old ones were all words hence the space chance. I could get a list of latin syllables and mix it in there rather than latin words, if you want?

**Comment by Anewbe** [comment]
Sure.

**Comment by Belsima** [comment]
Added most of the more common syllables in latin.

File: code/modules/mob/language/generic.dm [hunk 1]
```diff
@@ -26,7 +26,9 @@
 	whisper_verb = "whispers"
 	key = "0"
 	flags = RESTRICTED
-	syllables = list("vol", "zum", "coo","zoo","bi","do","ooz","ite","og","re","si","ite","ish","ar","at","on","ee","east","ma","da", "rim")
+	syllables = list(
+"vol", "zum", "coo","zoo","bi","do","ooz","ite","og","re","si","ite","ish",
+"ar","at","on","ee","east","ma","da", "rim")
```
Comment (Anewbe) [inline code/modules/mob/language/generic.dm]: Why was this changed?

Comment (Belsima) [reply code/modules/mob/language/generic.dm]: Consistent with every other language this way.
